### PR TITLE
Bah vtlogoctlanding 19182

### PR DESCRIPTION
**Pull request by ksantiagoBAH**
## Description
As a Comparison Tool user, I need to see the VET TEC logo in the Comparison Tool so that I have a consistent experience while viewing VET TEC related materials.

https://github.com/department-of-veterans-affairs/vets.gov-team/issues/19182
## Testing done
Run in local environment and navigate to http://localhost:3001/gi-bill-comparison-tool/

## Screenshots
Screen Shot 2019-07-18 at 7 59 54 AM
Screen Shot 2019-07-18 at 7 59 07 AM

Screen Shot 2019-07-18 at 8 00 13 AM

## Acceptance criteria
The logo (supporting artifact 2) is displayed on the landing page based on the following criteria:
- [x] The VET TEC Logo is not displayed on page load.
- [x] The VET TEC Logo is not displayed when the user selects "Schools" as the answer to "Type of institution".
- [x] The VET TEC Logo is not displayed when the user selects "Employers (On-the-job training [OJT], Apprenticeships)" as the answer to "Type of institution".
- [x] The VET TEC logo is displayed when the user selects "VET TEC training providers" as the answer to "Type of institution".
- [x] The VET TEC logo is 85px high.

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Review by zurbergram — APPROVED** [review]
LGTM

**Review by sonntag-adam — APPROVED** [review]

**Comment by ksantiagomorgan6** [comment]
@department-of-veterans-affairs/frontend-review-group Could I get a code review on this at your earliest convenience. Thank you!

File: src/applications/gi/components/search/TypeOfInstitutionFilter.jsx [hunk 1]
```diff
@@ -35,9 +34,21 @@ class TypeOfInstitutionFilter extends React.Component {
...
             rel="noopener noreferrer"
           >
-            Learn More
+            Learn More)
+            <br />
           </a>
-          )
+          {this.props.category === 'vettec' && (
+            <span className="vads-u-margin-x--neg5">
+              {' '}
```
Comment (rianfowler) [inline src/applications/gi/components/search/TypeOfInstitutionFilter.jsx]: just want to point this out and make sure it's not an oversight

**Review by rianfowler — APPROVED** [review]
LGTM

File: src/applications/gi/components/search/TypeOfInstitutionFilter.jsx (hunk 1 above)
Comment (ksantiagomorgan6) [reply src/applications/gi/components/search/TypeOfInstitutionFilter.jsx]: It is not an oversight, needed it to style and render logo.

**Pull request merged**